site/README.md

```diff
@@ -15,12 +15,16 @@ This folder contains the source code of the devSwag website (hosted at http://de
     ```
     npm install gulp-cli -g
     ```
-1. Open a terminal in the `swag-for-dev/site` directory. Type
+1. Open a terminal in the `swag-for-dev/site` directory. To install the dependencies, type 
     ```
     npm install
     ```
-    into the terminal to install the dependencies. To start the webserver, type
+    To build the webserver, type
     ```
-    gulp
+    gulp build
```

Comment:
This build command is not needed. Calling only `gulp` will build the assets and serve it through a webserver. Calling `gulp build` will only build the assets, it does a production build.